crates/noirc_evaluator/src/ssa/context.rs

```diff
@@ -748,13 +749,14 @@ impl SsaContext {
         Ok(())
     }
 
-    pub fn acir(&self, evaluator: &mut Evaluator) -> Result<(), RuntimeError> {
+    pub fn acir(&self, evaluator: &mut Evaluator, show_output: bool) -> Result<(), RuntimeError> {
         let mut acir = Acir::default();
         let mut fb = Some(&self[self.first_block]);
         while let Some(block) = fb {
             for iter in &block.instructions {
                 let ins = self.instruction(*iter);
-                acir.acir_gen_instruction(ins, evaluator, self).map_err(RuntimeError::from)?;
+                acir.acir_gen_instruction(ins, evaluator, self, show_output)
+                    .map_err(RuntimeError::from)?;
```

Comment:
I think the whole `.map_err` should be unnecessary here. Can we remove it?

Reply:
`fn acir` returns a `RuntimeError` while `acir_gen_instruction` returns a `RuntimeErrorKind`. It seems necessary unless I am missing something?

Reply:
`?` already calls `Into::into` on the error that is returned

Reply:
ah got it